lib/ocpp/v201/charge_point.cpp

```diff
@@ -1929,8 +1929,8 @@ bool ChargePoint::is_evse_reserved_for_other(EvseInterface& evse, const IdToken&
         const ConnectorStatusEnum status =
             evse.get_connector(static_cast<int32_t>(i))->get_effective_connector_status();
         if (status == ConnectorStatusEnum::Reserved) {
-            const std::optional<CiString<36>> groupIdToken =
-                group_id_token.has_value() ? group_id_token.value().idToken : std::optional<CiString<36>>{};
+            const std::optional<CiString<255>> groupIdToken =
```

Comment:
TODO: find out implications for backwards compatibility with 2.0.1

Reply:
accept